src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java

```diff
@@ -15,6 +18,37 @@ public AdvancementListener(HCCorePlugin plugin) {
         this.plugin = plugin;
     }
 
+    @EventHandler
+    public void onBlockBreak(final BlockBreakEvent event) {
+        // Check if it's a diamond ore
+        if (event.getBlock().getType() != Material.DIAMOND_ORE) {
+            return;
+        }
+
+        // The diamond ore wasn't found underground
+        final int MAX_Y_DIAMOND_ORE = 16;
+        if (event.getBlock().getY() > MAX_Y_DIAMOND_ORE) {
+            return;
+        }
+
+        Player player = event.getPlayer();
+        NamespacedKey key = new NamespacedKey(this.plugin, "mine_diamond_ore");
+        AdvancementProgress progress =
+                player.getAdvancementProgress(player.getServer().getAdvancement(key));
+        // Skip if player already has this advancement
+        if (progress.isDone()) {
+            return;
+        }
+
+        this.grantAdvancement(player, key);
+        player.sendMessage(ChatColor.GREEN
+                + "Congrats, you’ve found your very first diamond! You are now eligible for the exclusive Hack Club Minecraft Sticker—head over to "
```

Comment:
I would go with "stickers" instead of "sticker" - makes it sound better and it's true, given that there are 2-3 stickers in the pack.

Reply:
Changed in https://github.com/ifvictr/HCCore/pull/38/commits/66542e664218ddfc641c2757a6d53576301193df